Reply/app/src/main/java/com/materialstudies/reply/ui/email/EmailFragment.kt

```diff
@@ -62,6 +65,23 @@ class EmailFragment : Fragment() {
         savedInstanceState: Bundle?
     ): View? {
         binding = FragmentEmailBinding.inflate(inflater, container, false)
+
+        lifecycleScope.launch {
+            EmailUtils.selectedEmailState.collect { id ->
+                emailId = id
+                binding.email = EmailStore.get(id)
+            }
+        }
+
+        lifecycleScope.launch {
+            AdaptiveUtils.screenSizeState.collect { size ->
+                // Remove Navigation Icon in Large/XLarge screens
+                if (size == LARGE || size == XLARGE) {
+                    binding.toolbar.navigationIcon = null
```

Comment:
I think you'll want to re-set the navigationIcon if the size goes from LARGE back to SMALL. Like:

```
            AdaptiveUtils.screenSizeState.collect { size ->
                val navIcon = if (size == LARGE || size == XLARGE) {
                    null
                } else {
                    ContextCompat.getDrawable(requireContext(), R.drawable.ic_arrow_back_protected)
                }
                binding.toolbar.navigationIcon = navIcon
            }
```